src/client/datascience/api/jupyterIntegration.ts

```diff
@@ -97,7 +103,8 @@ export class JupyterExtensionIntegration {
                 product: Product,
                 resource?: InterpreterUri,
                 cancel?: CancellationToken
-            ): Promise<InstallerResponse> => this.installer.install(product, resource, cancel)
+            ): Promise<InstallerResponse> => this.installer.install(product, resource, cancel),
+            getDebuggerPath: async () => join(EXTENSION_ROOT_DIR, 'pythonFiles', 'lib', 'python')
```

Comment:
```suggestion
            getDebuggerPath: async () => dirname(getDebugpyPackagePath())
```
Just a suggestion, since this will ensure you get the parent directory of where  `debugpy` is. This way if we do move it as we decide to support more wheels types (this was in the some discussions) this will continue to work across the DS split.